### DISCUSSION TITLE
Why Discord over Discourse?

### DISCUSSION THREAD

**Discussion by lucasjamar** [kedro-org/kedro#1184] [Idea]
Hey Kedro Team!

I was wondering, but why was discourse shut down in favor of discord? Personally, I preferred discourse as the topics were divided into threads making google searching for a particular topic much easier. You also did not need to have an account so sharing information with colleagues was easier since they did not need to have an account. Sorry to be frank, but I had the impression the decision was taken without taking into consideration users' opinions (correct me if I'm wrong). Its also kind of sad that a lot of subjects had been tackled in discourse and that information is now gone.

**Comment by datajoely** [comment]
Hi @lucasjamar those are fair criticisms - I'll try to explain our thinking on this.

The short answer to this is - we consolidated our support channels into fewer sources we have to check. Our 8 or so strong maintainer group was somewhat struggling to support the various channels we had available at the time including GitHub Issues, StackOverflow, Discourse and more. We currently have no full time DevRel (if anyone is interested get in touch!) and the maintainer group was wary of the added support burden in discord in particular would bring. 

We deprecated discourse at the same time we we launched our [discord server](https://discord.gg/akJDeVaxnB) and [github discussions pages](https://github.com/kedro-org/kedro/discussions).  This leaves us with two officially supported channels of GitHub and Discord with StackOverflow 'unofficially' supported via a webhook that posts questions tagged with `#kedro` into our `#advanced-need-help` channel. 

Now I completely understand that the knowledge in discourse is gone and in retrospect we should have explored a mechanism for keeping the old conversations available in some sort of archive mode. You are also right we did make this decision with minimal user discussions and in this regard **we can do better and will do better in the future**. 

For now GitHub and Discord are our officially supported channels and are as not going anywhere fast.  Beyond that we're keen to hear any ways we can make the GitHub experience any better - from my point of view I've been excited with the amount of new features they have been adding / announced for both [discussions](https://github.com/github/feedback/discussions/categories/discussions-feedback) and [issues](https://github.blog/2021-06-23-introducing-new-github-issues/) and feel it's the right horse to back. Now we're part of the Linux Foundation you should see us transition to a much more transparent developer workflow as we move some of the last few systems and processes over.

**Reply by lucasjamar** [reply]
Hi @datajoely 

Thanks a lot for this answer. It's good to have an idea of the direction in which the kedro community is going :) .
